tags/find/find.py

```diff
@@ -1,16 +1,27 @@
-from talon import Module
+from talon import Context, Module, actions
 
 mod = Module()
 mod.tag("find", desc="Tag for enabling generic find commands")
 
+ctx = Context()
 
-@mod.action_class
-class Actions:
-    def find(text: str):
-        """Finds text in current editor"""
+ctx_mac = Context()
+ctx_mac.matches = r"""
+os: windows
+"""
 
-    def find_next():
-        """Navigates to the next occurrence"""
 
-    def find_previous():
-        """Navigates to the previous occurrence"""
+@ctx.action_class("edit")
+class EditActions:
+    def find(text: str = None):
+        actions.key("ctrl-f")
+        if text:
+            actions.insert(text)
+
+
+@ctx_mac.action_class("edit")
+class MacEditActions:
+    def find(text: str = None):
```

Comment:
Assuming this is meant to target macOS, the only correct macOS generic implementation exists in `edit_mac.py`.

Reply:
Okay I think we move these implementations to edit instead